### DISCUSSION TITLE
Very happy to have discovered Super Slicer ! but first print on a CR10S Pro did not go very well :(

### DISCUSSION THREAD

**Discussion by DzzD** [supermerill/SuperSlicer#1803] [General]
I have discovered Super Slicer on a video from youtube chanel "Teaching Tech" and was very happy to discover something that may really compete against Simplify3D ( that I was using since I started 3D printing ). 

Now I am not using S3D any more :) as Super Slicer is a lots more powerfull, as easy to use than S3D and give gcode that is as fast to print than S3D.  But I have to say that I have to really fight on my fist print to get a good result, so below is my fisrt experience story with SuperSlicer and tips to resolve problems for CR10S Pro users :
 
First I downloaded the Benchy STL from https://www.3dbenchy.com/download/ and try to print it on Simplify3D and SuperSlicer, the result was perfect with Simplify 3D and really bad with SuperSlicer, so after searching and testing for a while I finally found that there is a setting on the defaults CR10SPro printer profile that is IMO a lot too low, it is "minimum extrusion length": 

![image](https://user-images.githubusercontent.com/12198339/140605180-dfbd48b1-cae5-4f54-90c4-f5dbda63b769.png)

With this setting, the CR10S Pro cant handle high definition models (the Benchy have some parts that have a lot of tiny faces expecially on the front of the hull) the result is that the printer cant handle the gcode flow rate and it produce bad print. The solution was to set this setting to 0.25 and now everything is going well.

![image](https://user-images.githubusercontent.com/12198339/140605369-986e638c-1f87-42fc-abf6-8c9a8a6dafb1.png)

It should be interresting to change this default settings in SuperSlicer  for CR10SPro printer profile ?

Note : even for other printers I dont think a minimum extrusion length of 0.035 have a any sense as it is less than a tenth of most nozzle used (0.4)

Edit: another setting that can cause artifact in some cases is "infill/perimeter overlap", you may try to lower it is you encounter problems on smoothness of external faces

![image](https://user-images.githubusercontent.com/12198339/140605751-4d9b6553-816c-46fb-af7b-292852f83869.png)

**Comment by supermerill** [comment]
Thank you for the report.
In the next stable release, there will be a better setting than "minimum extrusion length", to take into account the speed.
I don't really know the best value to put into the default profile yet, but I will update them.
Do you have a little object, quick to print that shows the issue of nozzle pause?

**Comment by DzzD** [comment]
With a too low "minimum extrusion length" the problem appear all along the front of the Benchy hull, so you can just print the first 5mm height of the Benchy (joined to this post) to see the problem occur.

![image](https://user-images.githubusercontent.com/12198339/140610396-71c2a79b-7e80-45fe-bd02-c07db61d3899.png)

[3DBenchy.zip](https://github.com/supermerill/SuperSlicer/files/7490486/3DBenchy.zip)
